.gitignore

```diff
@@ -1,10 +1,643 @@
+# If this isn't enough to cover all your frameworks, editors, and IDEs, then I have no idea what you are doing
```

Comment:
You already have been told in https://github.com/nyaadevs/nyaa/pull/154#issuecomment-303216953 that most of these don't apply to this project or are better suited for the developer's global configuration.